server/routes/futureGoalTargetDateCalculator.test.ts

```diff
@@ -8,30 +8,30 @@ describe('futureGoalTargetDateCalculator', () => {
     // When
     const actual = futureGoalTargetDateCalculator(today, 3)
     // Then
-    expect(actual).toEqual({ value: '2024-05-26', text: 'in 3 months (26 May 2024)' })
+    expect(actual).toEqual({ value: '2024-05-26', text: 'in 3 months (by 26 May 2024)' })
```

Comment:
These changes reflect a very subtle change in the screen design content that was only communicated via the screenshots in the ticket! 🙄